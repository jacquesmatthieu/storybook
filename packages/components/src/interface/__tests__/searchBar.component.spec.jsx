import React from 'react';
import { shallow } from 'enzyme';

import SearchBar from '../searchBar.component';

describe('SearchBar', () => {
  let props;

  beforeEach(() => {
    props = {};
  });

  const getWrapper = () => shallow(<SearchBar {...props} />);

  it('should render properly', () => {
    // When
    const wrapper = getWrapper();

    // Then
    expect(wrapper).toMatchSnapshot();
  });
});
