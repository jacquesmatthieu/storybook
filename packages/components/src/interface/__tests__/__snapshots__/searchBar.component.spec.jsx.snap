// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar should render properly 1`] = `
<searchBarcomponent__StyledWrapper>
  <searchBarcomponent__StyledSearchBar
    variant="input"
  >
    <Icon
      fill="grey.dark"
      name="search"
      size={20}
      theme={
        Object {
          "palette": Object {},
        }
      }
    />
    <searchBarcomponent__StyledSearchInput
      onChange={[Function]}
    />
  </searchBarcomponent__StyledSearchBar>
</searchBarcomponent__StyledWrapper>
`;
